build_patch/apt-macos/path.diff

```diff
@@ -6,7 +6,7 @@ diff -urN apt-2.2.0/apt-pkg/init.cc apt/apt-pkg/init.cc
 
 
 -   Cnf.CndSet("DPkg::Path", "/usr/sbin:/usr/bin:/sbin:/bin");
-+   Cnf.CndSet("DPkg::Path", "/opt/procursus/sbin:/opt/procursus/bin:/usr/sbin:/usr/bin:/sbin:/bin");
++   Cnf.CndSet("DPkg::Path", "@MEMO_PREFIX@@MEMO_SUB_PREFIX@/sbin:@MEMO_PREFIX@@MEMO_SUB_PREFIX@/bin:/usr/sbin:/usr/bin:/sbin:/bin");
```

Comment:
I don't think you need to change this at all since /opt/procursus will most likely always be the install path for Procursus on macOS. Also this patch will only compile for macOS so I really see no point in adding the variables here when this path will probably never change unless macOS significantly changes in the future

Reply:
Would it be better if we keep thid patch like this, then unconditionally compile it?

Reply:
Actually now I think about maybe this method should be consider cause of rootless stuff, but I will let @CRKatri decide that since he would know better

Reply:
Move this patch to `build_patch/apt/` and keep the `@MEMO_PREFIX@@MEMO_SUB_PREFIX@` stuff, but don't forget to add `@MEMO_PREFIX@/bin` and `@MEMO_PREFIX@/sbin`.